app/components/concept-page/content.ts

```diff
@@ -13,8 +15,14 @@ interface Signature {
 }
 
 export default class ContentComponent extends Component<Signature> {
+  @service declare analyticsEventTracker: AnalyticsEventTrackerService;
+
   @tracked currentProgressPercentage = 0;
 
+  get conceptUrl() {
+    return `https://app.codecrafters.io/concepts/${this.args.concept.slug}`;
```

Comment:
Let's place this in `Concept#url` instead? That way we can re-use it in other places if needed.